specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/workspace.json

```diff
@@ -1276,7 +1276,6 @@
           "type": "boolean",
           "x-ms-mutability": [
             "create",
```

Comment:
Is it intentional that this field couldn't read?

refs: https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-mutability